contracts/extensions/OneTxPayment.sol

```diff
@@ -18,24 +18,16 @@
 pragma solidity 0.5.8;
 pragma experimental ABIEncoderV2;
 
-import "./../ColonyAuthority.sol";
-import "./../ColonyDataTypes.sol";
-import "./../IColony.sol";
-import "./../IColonyNetwork.sol";
+import "../ColonyAuthority.sol";
+import "../ColonyDataTypes.sol";
+import "../IColonyNetwork.sol";
+import "./ColonyExtension.sol";
 
 
-contract OneTxPayment {
+contract OneTxPayment is ColonyExtension {
   bytes4 constant ADD_PAYMENT_SIG = bytes4(keccak256("addPayment(uint256,uint256,address,address,uint256,uint256,uint256)"));
   bytes4 constant MOVE_FUNDS_SIG = bytes4(keccak256("moveFundsBetweenPots(uint256,uint256,uint256,uint256,uint256,uint256,address)"));
 
-  IColony colony;
-  IColonyNetwork colonyNetwork;
-
-  constructor(address _colony) public {
```

Comment:
Mm that's fair, I hadn't realized the overhead was so expensive.

Reply:
Mm so overriding `install` doesn't work since that is called by the ExtensionManager, which does not know anything about the extension apart from the resolver address. But I've done it in a different (and mildly hacky, but I think acceptable way).

Reply:
Problem solved by setting the resolver _before_ calling `install`, which allows for the resolver's version to be called (with any custom behavior).